docs/references/subcommands/project/edit.md

```diff
@@ -0,0 +1,59 @@
+## `fossa project edit`
+
+`fossa project edit` allows you to edit a FOSSA project's settings and configuration.
+
+## Options
+
+Argument                     | Required | Description
+-----------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------
+`--project-locator`          | Yes      | The project Locator defines a unique ID that the FOSSA API will use to reference this project within FOSSA. The project locator can be found in the UI on the project `Settings` page listed as the "Project Locator" underneath the "Project Title" setting.
```

Comment:
`--project-id` is not usable here. Hmm do you this would be confusing as a user? Based on the documentation in this doc, I thought that it was clear that `--project-locator` maps to `project.locator` in the fossa.yml configuration. I'm not opposed to specifying this in `.fossa.yml`  but just wanted to hear your opinion.

Reply:
I don't think the mapping is confusing, more that I could see someone trying to use project-id if they're used to using that already. I think that's fine though.